sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1093,6 +1093,12 @@ object RapidsConf {
     .toSequence
     .createWithDefault(Nil)
 
+  val LOG_TRANSFORMATIONS = conf("spark.rapids.sql.debug.logTransformations")
```

Comment:
should this be a log level instead of a Boolean config? Technically if the user changes the minimum log level to ERROR this setting has zero visible effect.

Reply:
I can see the argument for that, but I was really just following how we do other logging based on configuration options.